File: deviceinfo/src/main/java/com/an/deviceinfo/usercontacts/UserContacts.java
package com.an.deviceinfo.usercontacts;


import org.json.JSONObject;

import java.io.Serializable;

public class UserContacts implements Serializable {

    private String displayName;
    private String mobileNumber;
    private String phoneType;

    public String getDisplayName() {
        return displayName;
    }

    public void setDisplayName(String displayName) {
        this.displayName = displayName;
    }

    public String getMobileNumber() { return mobileNumber; }

    public void setMobileNumber(String mobileNumber) { this.mobileNumber = mobileNumber; }

    public String getPhoneType() { return phoneType; }

    public void setPhoneType(String phoneType) { this.phoneType = phoneType; }

    public JSONObject toJSON() {
        JSONObject jsonObject = new JSONObject();
        try {
            jsonObject.put("displayName", getDisplayName());
            jsonObject.put("mobileNumber", getMobileNumber());
            jsonObject.put("phoneType", getPhoneType());
            return jsonObject;

        } catch (Exception e) {
            e.printStackTrace();
            return null;
        }
    }
}

File: deviceinfo/src/main/java/com/an/deviceinfo/usercontacts/UserContacts.kt
package com.an.deviceinfo.usercontacts


import org.json.JSONObject

import java.io.Serializable

class UserContacts : Serializable {

    var displayName: String? = null
    var mobileNumber: String? = null
    var phoneType: String? = null

    fun toJSON(): JSONObject? {
        val jsonObject = JSONObject()
        try {
            jsonObject.put("displayName", displayName)
            jsonObject.put("mobileNumber", mobileNumber)
            jsonObject.put("phoneType", phoneType)
            return jsonObject

        } catch (e: Exception) {
            e.printStackTrace()
            return null
        }

    }
}